### ISSUE TITLE
Bug: Dynamic health and "Coin" health type

### ISSUE THREAD

**Issue by Tiburones202** [wofsauge/External-Item-Descriptions#724]
"{{HealingRed}} Heals {1} heart{2}" in "Red to Coin" in "RedToX" does not seem to replace the word "heart" to "coin"

Examples:
The "Heals 2 hearts" line in PHD does not change the word heart to coin (the icon changes tho)
The "Heals 4 hearts" line in Blood Bag does not change the word heart to coin

Extra:
Items such as Charm of the Vampire or The Gamekid still say "heart" instead of "coin", but I think that's not as important

**Comment by magagnato4106** [comment]
That's because they probably only attached the RedToX for the Health-ups. They need to extend it for everything that heals.

**Comment by Buurazu** [comment]
Yeah, it uses a data table for how many containers a Health Up item gives to know what specific numbers to look for in the description text.
PHD and other non-health-up healing items don't have a number assigned to look for, and then blood bag is an outlier because its health up amount is different than its healing amount. It still finds the {{HealingRed}} icon.
Although, it could be hardcoded to look for Heals 2, Heals 3, Heals 4 separately rather than just Heals {1} (same with gamekid/charm text)
